src/noflo-server.js

```diff
@@ -66,14 +67,16 @@ Promise.resolve(program.graph).then(function(filename){
         } else return noflo.graph.createGraph(filename);
     });
 }).then(function(graph){
-    return runtime(server, {
+    var rt = runtime(server, {
         defaultGraph: graph,
         baseDir: baseDir,
         captureOutput: false,
         catchExceptions: true,
         permissions: permissions,
         defaultPermissions: defaultPermissions
```

Comment:
Right now, I don't think we have any special permissions, yes?  Do we have any requirements on that, @dbooth-boston ?